hardhat.config.ts

```diff
@@ -1,45 +1,62 @@
-import { HardhatUserConfig } from "hardhat/config";
-import "@nomicfoundation/hardhat-toolbox";
-import "@typechain/hardhat";
-import "@nomicfoundation/hardhat-ethers";
-import "@nomicfoundation/hardhat-chai-matchers";
-import "dotenv/config";
+import { HardhatUserConfig } from 'hardhat/config';
+import '@nomicfoundation/hardhat-toolbox';
+import '@typechain/hardhat';
+import '@nomicfoundation/hardhat-ethers';
+import '@nomicfoundation/hardhat-chai-matchers';
+import 'dotenv/config';
 
 const config: HardhatUserConfig = {
-  // defaultNetwork: "polygon_amoy",
+  // defaultNetwork: 'gate_testnet',
 
   etherscan: {
     apiKey: {
-      lorenzo_testnet: "abc",
-    },
-    customChains: [
-      {
-        network: "lorenzo_testnet",
-        chainId: 83291,
+      // lorenzo_testnet: "abc",
 
-        urls: {
-          browserURL: "https://scan-testnet.lorenzo-protocol.xyz/",
-          apiURL: "https://scan-testnet.lorenzo-protocol.xyz/api/",
-        },
-      },
-    ],
+      // gate_testnet:"9063a1a8a3ee226fa440779bc6f7015a",
+      // bscTestnet:`CC7GINMBKSKVQZTXC9GD7M852K65CPEVRF`,
+      // polygon_amoy: `WIXGNHX39425CPDKS5TFBDJBKMGTDWR7QI,`,
+      // scroll_testnet: `a544040d-864b-4968-8829-12e160e66fb8`
+    },
   },
   networks: {
     lorenzo_testnet: {
-      url: "https://rpc-testnet.lorenzo-protocol.xyz",
+      url: 'https://rpc-testnet.lorenzo-protocol.xyz',
       accounts: process.env.PRIVATE_KEY ? [process.env.PRIVATE_KEY] : [],
     },
     polygon_amoy: {
-      url: "https://rpc-amoy.polygon.technology",
+      url: 'https://rpc-amoy.polygon.technology',
+      chainId: 80002,
+      accounts: [
+        'a0b8d309dd0afc95c94fc319f2de434f845a747d662c71ccbc08655c29299519',
+      ],
+      gasPrice: 25000000000, // 25 gwei in wei
+    },
+    gate_testnet: {
+      url: 'https://meteora-evm.gatenode.cc',
+      chainId: 85,
+      accounts: [
+        'ea5fbf909b888a1e810ef6ca42f06cb7eabc9a04095ec7e11fdeed7bb968ab07',
+      ],
+    },
+    bsc_testnet: {
+      url: 'https://data-seed-prebsc-2-s3.binance.org:8545/',
+      chainId: 97,
+      accounts: [
+        'ea5fbf909b888a1e810ef6ca42f06cb7eabc9a04095ec7e11fdeed7bb968ab07',
+      ],
+    },
+    scroll_testnet: {
+      url:'https://scroll-sepolia.drpc.org',
+      chainId:534351,
       accounts: [
-        "7c5b27c4f043051e405d03469e3f9dfe5b65df74376dcaf70db003d63a976efc",
+        'a0b8d309dd0afc95c94fc319f2de434f845a747d662c71ccbc08655c29299519',
```

Comment:
Why do you expose private keys here? You can use process.env.PRIVATE_KEY just like it's used in other places: `accounts: process.env.PRIVATE_KEY ? [process.env.PRIVATE_KEY] : []`